config/perf_configs/flame_perf_target_prognostic_edmfx_aquaplanet.yml

```diff
@@ -2,6 +2,7 @@ job_id: flame_perf_target_prognostic_edmfx_aquaplanet
 vert_diff: "true" 
```

Comment:
```suggestion

```

Reply:
Oh, I meant to set it to false lol. (I have to set it because the default perf config is true.)